app/assets/javascripts/app/views/people.coffee

```diff
@@ -0,0 +1,28 @@
+PersonView = Backbone.View.extend {
+  className: 'card'
+  template: _.template "<img class='image' src='<%= imagePath %>'></img>" +
+    "<span class='name'><%= name %></span>" +
+    "<span class='title'><%= title %></span>" +
+    "<span class='bio'><%= bio %></span>"
+  render: ->
+    @$el.html @template(@model.toJSON())
+    this
+}
+
+person = new Person
+peopleList = new PeopleList
+
+PeopleListView = Backbone.View.extend {
+  collection: peopleList
+  render: ->
+    @collection.forEach(@addOne, this)
+    this
+  addOne: (person) ->
+    personView = new PersonView {model: person}
```

Comment:
here's where that variable is called. looking at it all together, is this a correct usage?